packages/prerender-fargate/lib/prerender/Dockerfile

```diff
@@ -4,19 +4,18 @@ ENV CHROME_BIN=/usr/bin/chromium-browser
 ENV CHROME_PATH=/usr/lib/chromium/
 ENV MEMORY_CACHE=0
 
-# install chromium, tini and clear cache
-RUN apk add --update-cache chromium tini \
- && rm -rf /var/cache/apk/* /tmp/*
+RUN apk add --no-cache chromium tini
 
 USER node
 WORKDIR "/home/node"
 
 COPY ./package.json .
-COPY ./server.js .
 
-# install npm packages
 RUN npm install --no-package-lock
 
+COPY ./server.js .
+COPY ./utils ./utils
```

Comment:
Minor but I think these should be swapped around, we're more likely to modify the server.js file

Reply:
Updated